src/condor_collector.V6/collector_engine.cpp

```diff
@@ -621,7 +621,7 @@ collect (int command,ClassAd *clientAd,const condor_sockaddr& from,int &insert,S
 				// the startd could stop bothering to send these attributes.
 
 				// Queries of private ads depend on the following:
-			SetMyTypeName( *pvtAd, STARTD_ADTYPE );
+			CopyAttribute(ATTR_MY_TYPE, *pvtAd, *clientAd);
```

Comment:
For the convenience of any future reviewers, this is actually indented correctly, but looks like it isn't because of a silly comment-placement convention.